src/protocols/aeternity/views/TransactionDetails.vue

```diff
@@ -1,54 +1,60 @@
 <template>
-  <div class="transaction-details">
-    <Loader v-if="!transaction || transaction.incomplete" />
-    <template v-else>
-      <TransactionDetailsBase
-        :transaction="transaction"
-        :coin-symbol="AE_SYMBOL"
-        :transaction-fee="+aettosToAe(transactionFee)"
-        :is-swap="isSwap"
-        :is-pool="isPool"
-        :token-symbol="getTxSymbol(transaction)"
-        :total-amount="getTxAmountTotal(transaction, direction)"
-        :is-error-transaction="isErrorTransaction"
-        :is-dex-allowance="isDexAllowance"
-        :is-dex="isDex"
-        :is-transaction-aex9="isTransactionAex9(transaction)"
-        :payload="getTransactionPayload(transaction)"
-        :is-multisig="isMultisig"
-        :direction="direction"
-        :explorer-url="explorerUrl || ''"
-        :is-local-account-address="isLocalAccountAddress"
-        :gas-price="gasPrice ? +aettosToAe(gasPrice) : 0"
-        :gas-used="gasUsed"
-        :contract-id="contractId"
-        :multisig-transaction-fee-paid-by="multisigTransactionFeePaidBy"
-        :multisig-contract-id="multisigContractId"
-        :hash="hash"
-        :protocol="PROTOCOL_AETERNITY"
-      >
-        <template #tip-url>
-          <DetailsItem
-            v-if="tipUrl"
-            :label="$t('pages.transactionDetails.tipUrl')"
-            class="tip-url"
-            data-cy="tip-url"
+  <ion-page>
+    <ion-content
+      class="ion-padding"
+    >
```

Comment:
1. Please use PascalCase for component naming to maintain consistency.
2. I don't think splitting those lines is necessary.